_docs-sources/library/overview/modules.md

```diff
@@ -31,3 +51,9 @@ The code in the module repos are organized into three primary folders:
 1. `test`: Automated tests for the code in modules and examples.
 
 We follow Hashicorp's [Standard Model Structure](https://developer.hashicorp.com/terraform/language/modules/develop/structure) for our files (`main.tf`, `variables.tf`, `outputs.tf`). In the `variables.tf` file we always put the required variables at the top of the file, followed by the optional variables. Although there are often a lot of ways to configure our modules, we set reasonable defaults and try to minimize the effort required to configure the modules to the most common use cases.
+
+## How modules are updated
+
+Gruntwork brings together AWS and Terraform experts around the world who track updates from AWS, Terraform, and the DevOps community at large, along with requests from the Gruntwork customer community. We translate the most important of these updates into new features, new optimizations, and ultimately new releases.
```

Comment:
This describes an abstract concept of how we update. We should link to the page that describes it in detail. `/library/overview/modules#how-modules-are-updated`

Reply:
Fair point, though I don't see the page you're referring to? The path you listed refers to this section, and I couldn't find any other pages so will leave for now.